src/markdown-pages/build-apps/customize-open-source-visualization.mdx

```diff
@@ -0,0 +1,1141 @@
+
+- [colorScale](https://formidable.com/open-source/victory/docs/victory-pie/#colorscale)
+- [cornerRadius](https://formidable.com/open-source/victory/docs/victory-pie/#cornerradius)
+
+<Steps>
+
+<Step>
+
+In your local Nerdpack, open _nr1-victory-visualizations/visualizations/circular-progress-bar/nr1.json_.
+
+_nr1.json_ is the **Circular progress bar** visualization's [metadata file](http://localhost:8000/explore-docs/custom-viz/configuration-options/). Use this file to add a configurable `colorScale` option, which corresponds to the `colorScale` field on `VictoryPie`.
+
+</Step>
+
+<Step>
+
+Add a [`collection`](/explore-docs/custom-viz/configuration-options/#collection) of [`string`](/explore-docs/custom-viz/configuration-options/#string) fields for you to customize your chart's colors:
+
+```json fileName=nr1.json
+{
+  "schemaType": "VISUALIZATION",
+  "id": "circular-progress-bar",
+  "displayName": "Circular progress bar",
+  "description": "",
+  "configuration": [
+    {
+      "name": "nrqlQueries",
+      "title": "NRQL Queries",
+      "type": "collection",
+      "items": [
+        {
+          "name": "accountId",
+          "title": "Account ID",
+          "description": "Account ID to be associated with the query",
+          "type": "account-id"
+        },
+        {
+          "name": "query",
+          "title": "Query",
+          "description": "NRQL query for visualization",
+          "type": "nrql"
+        }
+      ]
+    },
+    {
+      "name": "thresholds",
+      "title": "Thresholds",
+      "type": "namespace",
+      "items": [
+        {
+          "name": "criticalThreshold",
+          "title": "Critical threshold",
+          "description": "Value at which progress is displayed as critical",
+          "type": "number"
+        },
+        {
+          "name": "highValuesAreSuccess",
+          "title": "Above threshold is success",
+          "description": "If toggled on, values above the threshold display as successful. Otherwise, values at or above the threshold display as critical.",
+          "type": "boolean"
+        }
+      ]
+    },
+    {
+      "name": "colors",
+      "title": "Colors",
+      "type": "collection",
+      "items": [
+        {
+          "name": "segmentColor",
+          "title": "Segment color",
+          "description": "The color of a bar segment.",
+          "type": "string"
+        }
+      ]
+    }
+  ]
+}
+```
+
+The `VictoryPie` field that you'll use with this update is called [`colorScale`](https://formidable.com/open-source/victory/docs/victory-pie/#colorscale). It accepts an array of colors and applies each color to a segment of the progress bar. So, in your visualization's configuration options, you've specified a collection of strings that you use to pass colors to your chart.
+
+</Step>
+
+<Step>
+
+In the same visualization directory, open _index.js_.
+
+</Step>
+
+<Step>
+
+In `render()`, set the `VictoryPie.colorScale` value:
```

Comment:
The dot notation here feels a bit confusing. Could these "step header" sections be revised to something like this?

"In render(), define VictoryPie's colorScale prop:"